tools/downloader/quantizer.py

```diff
@@ -27,23 +27,36 @@
 
 import common
 
+import yaml
```

Comment:
Please, move it to rest core imports.

Reply:
I did mess up the import order (which I have now fixed), but note that `yaml` is a third-party module, so I'm not grouping it with the standard library modules.